components/pages/privacy/access/requests/index.jsx

```diff
@@ -87,73 +92,66 @@ export default function ConsentShow() {
   if (!consentRequest) return <Spinner />;
 
   return (
-    <ConsentRequestProvider
-      consentRequest={consentRequest}
-      setConsentRequest={setConsentRequest}
-    >
-      <MuiPickersUtilsProvider utils={DateFnsUtils}>
-        <Container className={bem("request-container")}>
-          <ConsentRequestForm
-            agentDetails={agentDetails}
-            agentWebId={agentWebId}
-          />
-          <div className={bem("request-container__content")}>
-            <Typography
-              component="h3"
-              align="center"
-              variant="h3"
-              className={bem("heading__uppercase")}
-            >
-              About this application
-            </Typography>
-            <div className={bem("app-name")}>
-              <Icons className={bem("avatar")} name="project-diagram" />
-              <Typography align="center" variant="body2">
-                <span className={bem("footer__link")}>
-                  <Link href={agentWebId} variant="body2">
-                    {agentName || agentWebId}
-                  </Link>
-                </span>
-              </Typography>
-            </div>
-            <Typography className={bem("footer__links")}>
-              {agentUrl && (
-                <span className={bem("footer__link")}>
-                  <Link href={agentUrl} variant="body2">
-                    <Icons
-                      name="globe"
-                      className={bem("icon-small", "primary")}
-                    />
-                    Website
-                  </Link>
-                </span>
-              )}
-              {agentPolicy && (
-                <span className={bem("footer__link")}>
-                  <Link href={agentPolicy} variant="body2">
-                    <Icons
-                      name="webid"
-                      className={bem("icon-small", "primary")}
-                    />
-                    Privacy Policy
-                  </Link>
-                </span>
-              )}
-              {agentTOS && (
-                <span className={bem("footer__link")}>
-                  <Link href={agentTOS} variant="body2">
-                    <Icons
-                      name="doc"
-                      className={bem("icon-small", "primary")}
-                    />
-                    Terms of Service
-                  </Link>
-                </span>
-              )}
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <Container className={bem("request-container")}>
+        <ConsentRequestForm
+          consentRequest={consentRequest}
+          agentDetails={agentDetails}
+          agentWebId={agentWebId}
+        />
+        <div className={bem("request-container__content")}>
```

Comment:
It might be nice to move this to an `AgentDetails` component that takes in `agentDetails` and `agentWebId`, though not strictly necessary (but I think this UI might reoccur through out the application)